cloud-shell/src/index.ts

```diff
@@ -67,6 +66,9 @@ rpcConnecton.create().then(connection => {
             attachConnection.onclose = (event: CloseEvent) => {
                 console.log('Attach connection closed: ', event.code);
             }
+        }, 
+        (onRejected: any) => {
+            terminal.sendLine(CS.RED_COLOR + 'Unable to connect to json-rpc channel. Cause: ' + onRejected + CS.RESET_COLOR)
```

Comment:
Does it make sense to leave logging here as well? At least debug level

Reply:
@sleshchenko it is the place where is we receive error, but not enough permissions and so on. You created issue about login this error...

Reply:
Ah, don't mind. I did not give enough attention to understand that it's a client-side code. So, updating UI without writing browser logs should be enough